populate-taxidermy-content.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+///INDEX POSITION//
+function populateItems(array $results) :string {
+    $collectionString = '';
+    foreach ($results as $row) {
+        $collectionString .= populateItem($row);
```

Comment:
I think you're not supposed to call functions inside other functions.
There's usually a way to call it outside of the function then pass the returned result to the first function as a variable.